src/main/__tests__/helpers/testHelpers.ts

```diff
@@ -79,7 +44,22 @@ export async function expectIframeWithParams(
   } else fail('No iframe found!')
 }
 
-export const mockPkceWindow = {
+export async function expectIframeWithParamss(
```

Comment:
Typo `Params`

Reply:
Thanks, good catch, it was initially because I had the old `expectIframeWithParams` implementation cohabiting with this one.